### ISSUE TITLE
Improvements to the 1st tutorial 

### ISSUE THREAD

**Issue by adam-dziedzic** [tensorflow/tensorflow#20167]
Regarding the very first tutorial: https://www.tensorflow.org/get_started/eager

You should add the import: 

import tensorflow.contrib.eager as tfe;

and change the types for example_default to:

example_defaults = [[0.], [0.], [0.], [0.], [0]]  # set field types (the labels are integers: https://stackoverflow.com/questions/50184145/tensorflow-sparsesoftmaxcrossentropywithlogits-error)

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by adam-dziedzic** [comment]
All of these fields are N/A. This is a general improvement of the code for people who copy/paste the code from the tutorial.

**Comment by bignamehyp** [comment]
I don't quite follow your suggestion.

**Comment by adam-dziedzic** [comment]
I see that it was updated.
